app/models/evaluation.rb

```diff
@@ -12,4 +12,69 @@ class Evaluation < ActiveRecord::Base
   def update_course_rating_and_count
     Course.update_course_rating_attributes course_id
   end
+
+  def self.collect_evaluation_objects_for_course(course)
+    if course.evaluations.present?
+      evaluations_from_previous_course = nil
+      course_evaluations = course.evaluations
+    elsif course.previous_iteration_id.present?
+      course_evaluations, evaluations_from_previous_course = Evaluation.collect_evaluations_from_a_previous_course_iteration(course)
```

Comment:
I have problems reading this line. Which value is assigned to which variable?

Just try this minimal working example. I think, it is the opposite of what you want:

```ruby
2.3.0 :001 > a, b = 'HI'
 => "HI" 
2.3.0 :002 > a
 => "HI" 
2.3.0 :003 > b
 => nil 
```